client/my-sites/marketing/buttons/buttons.jsx

```diff
@@ -159,12 +162,19 @@ class SharingButtons extends Component {
 					<Notice
 						status="is-warning"
 						showDismiss={ false }
-						text={ translate(
-							'Adding sharing buttons need the Sharing Buttons module from Jetpack to be enabled'
-						) }
+						text={
+							isPrivate
+								? translate( 'Adding sharing buttons requires your site to be marked as Public.' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.